src/covvfit/_preprocess_abundances.py

```diff
@@ -13,34 +14,56 @@ def preprocess_df(
     df: pd.DataFrame,
     cities: list[str],
     variants: list[str],
-    undertermined_thresh: float = 0.01,
+    *,
```

Comment:
I added "*", [making the rest of the arguments keyword-only](https://peps.python.org/pep-0570/#keyword-only-arguments), which hopefully will limit the number of bugs introduced by accidentally swapping `date_min` and `date_max` or column names.